questionnaire/questionnaire-service.js

```diff
@@ -350,7 +350,23 @@ function createQuestionnaireService({
         return progressEntryResource;
     }
 
+    function applicationSubmitted(submissionStatus) {
+        return submissionStatus !== 'NOT_STARTED';
+    }
+
     async function buildSessionInformationBlock(questionnaireId) {
+        const submissionStatus = await getQuestionnaireSubmissionStatus(questionnaireId);
+
+        if (applicationSubmitted(submissionStatus)) {
+            const dateNow = Date.now();
+            return {
+                alive: false,
+                duration: 0,
+                created: dateNow,
+                expires: dateNow
```

Comment:
not understanding the reason behind these 3 properties. duration should not be 0, and it was created before now.

Reply:
These properties represent the state of _this_ session resource, the duration is 0 seconds as the session has ended, the resource is being created and expired at the same time. In essence all parameters represent an invalid session expiry resource.

Reply:
happy to discuss also happy to change this from dateNow to creationDate or equivalent